<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: white;
	 border-radius: 25px;
}

QPushButton
{
    color: white;
    background-color: #1A4A8B;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
}

QPushButton:disabled
{
    color: #1A4A8B;
    background-color: white;
    border-radius: 0px;
}

QPushButton:Pressed
{
    background-color: white;
}

QLabel
{
	color: white;
    background-color: #1A4A8B;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
}

QRadioButton{
	font: 14pt &quot;Bahnschrift&quot;;
	color: #1A4A8B;
}

QLineEdit
{
	color: #1A4A8B;
    background-color: white;
    border : 2px solid #1A4A8B;
    border-radius: 15px;
    font: 10pt &quot;Bahnschrift&quot;;
    border: 1px solid #1A4A8B;
    padding: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:0px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">padding:10px;</string>
           </property>
           <property name="text">
            <string>Start Player</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="machine">
           <property name="styleSheet">
            <string notr="true">padding:20px;</string>
           </property>
           <property name="text">
            <string>Machine Start </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="human">
           <property name="styleSheet">
            <string notr="true">padding:20px;</string>
           </property>
           <property name="text">
            <string>Human Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">padding:10px;</string>
                 </property>
                 <property name="text">
                  <string>Algorithm</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="mm_original">
                 <property name="text">
                  <string>Original MinMax</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="mm_parallel1">
                 <property name="text">
                  <string>Parallel MinMax Ver 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="mm_parallel2">
                 <property name="text">
                  <string>Parallel MinMax Ver 2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">padding:10px;</string>
                 </property>
                 <property name="text">
                  <string>Search Tree Depth</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="dephtLe">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">padding:10px;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="save">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>44</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save Changes</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: #1A4A8B;
    background-color: white;
    border-radius: 15px;
    font: 9pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>Nota : Recuerde que elegir niveles de profundidad muy grandes aumentara el tiempo de respuesta exponencialmente,  el valor recomendado es 5</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
